### PR TITLE
Vboulos add vmware opp job

### PR DESCRIPTION
**Pull request by vboulos**
- Add vSphere ipi workflow
- Add new OPP job to run on vSphere and OCP 4.17

### CODE REVIEW

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

File: ci-operator/jobs/stolostron/policy-collection/stolostron-policy-collection-main-presubmits.yaml [hunk 1]
```diff
@@ -116,3 +116,61 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )opp-ocp4.16-interop-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (dhaiducek) [inline ci-operator/jobs/stolostron/policy-collection/stolostron-policy-collection-main-presubmits.yaml:120]:
```suggestion
    always_run: false
```

File: ci-operator/config/stolostron/policy-collection/stolostron-policy-collection-main__opp-ocp4.16-interop-vsphere.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- dockerfile_literal: |
+    FROM this-is-ignored
+    RUN dnf install -y git
+  from: cli
+  to: cli-with-git
```
Comment (dhaiducek) [inline ci-operator/config/stolostron/policy-collection/stolostron-policy-collection-main__opp-ocp4.16-interop-vsphere.yaml:44]:
```suggestion
  optional: true
  to: cli-with-git
```

**Review by dhaiducek — COMMENTED** [review]
The `images` presubmit jobs (i.e. building the image in PRs) should be set to not run.

I know for sure the `always_run` parameter works to accomplish this. Alternatively, I discovered the `optional` parameter in the `images` stanza, which, if that works, will be better since it wouldn't get overwritten when the jobs are updated.

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse  periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by dtthuynh** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.15-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.15-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.15-interop-openshift-plus-interop-aws

**Comment by dtthuynh** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.15-interop-openshift-plus-interop-aws

**Comment by dtthuynh** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.15-interop-openshift-plus-interop-aws

**Comment by dtthuynh** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.15-interop-openshift-plus-interop-aws

**Comment by dtthuynh** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.15-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.15-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.15-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.15-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by dtthuynh** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.16-interop-openshift-plus-interop-aws

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.17-interop-vsphere-openshift-plus-interop-vsphere

**Comment by vboulos** [comment]
/pj-rehearse periodic-ci-stolostron-policy-collection-main-opp-ocp4.18-interop-vsphere-openshift-plus-interop-vsphere

**Review by gparvin — APPROVED** [review]

**Pull request merged**